cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -127,13 +118,10 @@
   private static final String TYPE_RUN_RECORD_STARTED = "runRecordStarted";
   private static final String TYPE_RUN_RECORD_SUSPENDED = "runRecordSuspended";
   private static final String TYPE_RUN_RECORD_COMPLETED = "runRecordCompleted";
-  private static final String TYPE_WORKFLOW_NODE_STATE = "wns";
-  private static final String TYPE_WORKFLOW_TOKEN = "wft";
-  private static final String TYPE_NAMESPACE = "namespace";
-  private static final String TYPE_MESSAGE = "msg";
 
   private static final String TYPE_COUNT = "runRecordCount";
   private static final String TYPE_RUN_RECORD_UPGRADE_COUNT = "runRecordUpgradeCount";
+  private static final String SMALLEST_POSSIBLE_STRING = "";
```

Comment:
Why not call it EMPTY_STRING?

Reply:
SMALLEST_POSSIBLE_STRING is more descriptive imo since we're using it as the start of a scan